integration-test-jdbc/src/test/kotlin/integration/jdbc/JdbcDataTypeTest.kt

```diff
@@ -329,6 +329,28 @@ class JdbcDataTypeTest(val db: JdbcDatabase) {
         assertEquals(data, data2)
     }
 
+    @Test
+    fun clobString() {
+        val m = Meta.clobStringData
+        val data = ClobStringData(1, "abc")
```

Comment:
Would there be any way to have it return as a String here? Or maybe have some way to tell the dsl when to return which type... it's a bit inconsistent that in the the Entity the user wants it as a string, whereas when returning it as a single field it ends up being the underlying data class (which is really just a technical detail of how Komapper works, and not really what is expected by the user?

Reply:
I do not fully understand what you are saying. `ClobStringData` is an entity class. You can get a String from the `value` property. 
`ClobStringData` is defined as follows:
```kotlin
@KomapperEntity
@KomapperTable("clob_data")
data class ClobStringData(
    @KomapperId val id: Int,
    @KomapperColumn(alwaysQuote = true, alternateType = ClobString::class)
    val value: String?,
)
```

Reply:
Oh... I didn't pay 100% attention that you were retrieving the whole data class there... (`ClobString` and `ClobStringData` are a bit close... and I just went over that code a bit too quickly...). 

So what happens if the user requests `select(m.value)`, it returns a `String`? If so, then all this is just fine. I guess I was kinda expecting a test on that, but it's probably tested elsewhere...

Reply:
> So what happens if the user requests select(m.value), it returns a String?

Of course YES.